package.json

```diff
@@ -101,6 +102,10 @@
     "storybook:static:build": "build-storybook --config-dir config/storybook --output-dir build/storybook",
     "storybook:static:serve": "serve build/storybook --port 5555 || true",
     "storybook:static:stop": "pkill -f \"node_modules/.bin/serve build/storybook\"",
+    "svg:clean-build": "rimraf build/icons",
+    "svg:clean-components": "rimraf src/components/Icon/icons/*.tsx",
```

Comment:
I would move these two tasks up with the other `clean` tasks, and rename them `clean:icons` and `clean:components` maybe? That way all tasks that delete stuff are grouped together under the same prefix.